classifiers/test.py

```diff
@@ -46,15 +47,16 @@ def test_model(model, data_loader, loss_function, device):
     total_acc = current_acc.double() / len(data_loader.dataset)
 
     time_elapsed = time.time() - since
-    print('Test complete in {:.0f}m {:.0f}s'.format(time_elapsed // 60, time_elapsed % 60))
+    print('Resnet18 Test complete in {:.0f}m {:.0f}s'.format(time_elapsed // 60, time_elapsed % 60))
```

Comment:
It's better to use args.model here to avoid confusion in cases where we call this file for other classifiers